modules/administration/pages/auth-methods-sso-example.adoc

```diff
@@ -40,6 +40,76 @@ java.sso.onelogin.saml2.idp.single_logout_service.url
 java.sso.onelogin.saml2.idp.x509cert
 ----
 
+Here is an example of the final SUSE Manage's rhn.conf:
```

Comment:
```suggestion
Here is an example of [path]``rhn.conf`` on {productname} after enabling SSO:
```
It's a proposal based on the description of this PR.

Reply:
Looks good to me. Thanks Karl. 👍